utility.lua

```diff
@@ -85,6 +94,7 @@ hopper.take_item_from = function(hopper_pos, target_pos, target_node, target_inv
 	local placer = get_placer(hopper_meta:get_string("placer"))
 
 	--source inventory
+	local target_pos = real_inv_pos(target_pos)
```

Comment:
Unfortunately there's no luacheck action set up in this mod. If it was, this would cause an error because you're shadowing a variable in the same scope.
Suggestion: remove `local`. Same for L148.

Reply:
I didn't know about his existence. Thank you.